Documentation/Flashing.md

```diff
@@ -88,18 +88,15 @@ Check the extension of your firmware, it should be `.RDY` now.
 
 ## Pinecil (Pine64)
 
-The MCU used in the Pinecil supports usb-dfu.
+The MCU used in the Pinecil supports usb-dfu. Reference Pine64 [Wiki](https://wiki.pine64.org/wiki/Pinecil).
 
-There is the nice gui [Pinecil Firmware Updater](https://github.com/pine64/pinecil-firmware-updater/releases). In the event that this tool is unavailable or otherwise does not meet your needs, the below process may be used to update the firmware.
+There is the nice gui [Pinecil Firmware Updater](https://github.com/pine64/pinecil-firmware-updater/releases). It works in several types of OS, i.e. Windows/Mac. If your OS is not currently supported by the Updater or it does not meet your needs, the below manual process may be used to update the firmware.
 
-To enter DFU mode, hold the button at the back of the iron while connecting to usb over usb-c.
+To enter DFU mode, long hold the (-) button at the back of the iron, then connect to usb over usb-c.

```

Comment:
Your new wording does not indicate to continue holding it while connecting instead it sounds like you hold it down, let go, then connect USB 😅

Reply:
fixed